build.sbt

```diff
@@ -133,7 +140,7 @@ lazy val dependencySettings = Seq(
 lazy val mdocSettings = Seq(
   mdoc := (Compile / run).evaluated,
   scalacOptions --= Seq("-Xfatal-warnings", "-Ywarn-unused"),
-  crossScalaVersions := Seq(scalaVersion.value),
+  crossScalaVersions := Seq(scala213),
```

Comment:
sbt started complaining about a cycle between `crossScalaVersions` and `scalaVersion` when I added the sbt-typelevel plugin 🤷